guides/source/active_support_core_extensions.md

```diff
@@ -837,6 +839,25 @@ In the previous example the macro generates `avatar_size` rather than `size`.
 
 NOTE: Defined in `active_support/core_ext/module/delegation.rb`
 
+#### `delegate_missing_to`
+
+Imagine you would like to delegate all of the `Profile` methods to the `User`
+object. Instead of listing every `Profile` method in a `delegate` call, you can
```

Comment:
> delegate all of the `Profile` methods to the `User` object

That sounds around the wrong way -- though the exact opposite isn't right either.

'to' -> 'from' might be sufficient?